nservicebus/extending/queuecreation.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
also this would break the docs build since a directory needs to have an `index.md`